code/mob/living/carbon/human/npc.dm

```diff
@@ -640,9 +640,10 @@
 	. = ..()
 	if(!src.ai_active)
 		return
-	if(src.ai_state == AI_FLEEING && ai_incapacitated())
-		src.ai_state = AI_PASSIVE
-		walk_away(src, null)
+	if(ai_incapacitated())
+		walk(src, null)
```

Comment:
Equivalent to `walk_away(src, null)`. Reference says to use `walk()` specifically though.